src/features/user/Settings/EditProfile/AddressManagment/index.tsx

```
@@ -0,0 +1,189 @@
+            onClick={toggleAddAddressModal}
+            variant="primary"
+            buttonClasses={styles.addAddressButton}
+          />
+        ) : (
+          <p className={styles.maxAddress}>
+            {tAddressManagement('maxAddressesMessage')}
+          </p>
+        )}
+      </CenteredContainer>
+      <Modal open={isModalOpen}>
+        <>{renderModalContent}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)